### PR TITLE
[framework] introduced new phing target demo-data that wipes and completely installs demo data including export to Elasticsearch

### PR DESCRIPTION
**Pull request by TomasLudvik**
| Q             | A
| ------------- | ---
|Description, reason for the PR| introduced new phing target `demo-data` that wipes and completely installs demo data including export to Elasticsearch
|New feature| No 
|[BC breaks](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/)| No 
|Fixes issues| ... 
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

File: packages/framework/build.xml [hunk 1]
```diff
@@ -309,6 +311,8 @@
         </exec>
     </target>
 
+    <target name="demo-data" depends="production-protection,db-wipe-public-schema,db-import-basic-structure,db-migrations,domains-data-create,db-fixtures-demo,plugin-demo-data-load,friendly-urls-generate,domains-urls-replace,elasticsearch-export" description="Wipes Postgres DB, recreates structure, fills with demo data and then exports data to Elasticsearch"/>
```
Comment (pk16011990) [inline packages/framework/build.xml:314]: Now there is no separate target that will only allow running demo data. I see this as limiting for common scenarios.

The other thing is that with Postgres, structure creation and migrations are included, but not for Elasticsearch, which is not consistent.

Comment (TomasLudvik) [reply packages/framework/build.xml:314]: I find it correct, that there is no target for running demo data creation only in Postgres, as it leads into faulty application, as you show different data on frontend than the data present in database.

`phing elasticsearch-export` does also migrations if needed see `IndexFacade::export` for more information.

Comment (grossmannmartin) [reply packages/framework/build.xml:314]: I would also appreciate the separate (even hidden) target for load data to database only. 
I use it on purpose when I know I don't need elastic. 

But the final decision is yours of course.

Comment (TomasLudvik) [reply packages/framework/build.xml:312]: I have hide db-demo, but kept it as it was

**Review by pk16011990 — APPROVED** [review]

**Pull request merged**